data/README.md

```diff
@@ -41,6 +41,11 @@
 | bloom_filter.bin                               | deprecated bloom filter binary with binary header and murmur3 hashing |
 | bloom_filter.xxhash.bin                        | bloom filter binary with thrift header and xxhash hashing    |
 | nan_in_stats.parquet                           | statistics contains NaN in max, from PyArrow 0.8.0. See note below on "NaN in stats".  |
+| rle-dict-snappy-checksum.parquet                 | compressed INT32 and STRING columns in v2 dictionary pages with a matching CRC |
```

Comment:
Does it matter whether the file format is v1 or v2? It seems to me that they are the same dictionary pages except the encoding name. I'd suggest to add less files to keep this repo small.

Reply:
Okay, previously, I though we can test format-V1 and format-V2 files. But exactly, v2 is enough for us.